<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiled</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Tiled::Internal::MapView" name="mapView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectNone"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>&amp;Map</string>
    </property>
    <addaction name="actionNewTileset"/>
    <addaction name="separator"/>
    <addaction name="actionResizeMap"/>
    <addaction name="actionMapProperties"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowGrid"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomNormal"/>
   </widget>
   <widget class="QMenu" name="menuLayer">
    <property name="title">
     <string>&amp;Layer</string>
    </property>
    <addaction name="actionAddTileLayer"/>
    <addaction name="actionAddObjectLayer"/>
    <addaction name="actionDuplicateLayer"/>
    <addaction name="actionRemoveLayer"/>
    <addaction name="separator"/>
    <addaction name="actionMoveLayerUp"/>
    <addaction name="actionMoveLayerDown"/>
    <addaction name="separator"/>
    <addaction name="actionLayerProperties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuMap"/>
   <addaction name="menuLayer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-save.png</normaloff>:/images/16x16/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/application-exit.png</normaloff>:/images/16x16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-copy.png</normaloff>:/images/16x16/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-paste.png</normaloff>:/images/16x16/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/help-about.png</normaloff>:/images/16x16/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Tiled</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionResizeMap">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-page-setup.png</normaloff>:/images/16x16/document-page-setup.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Resize Map...</string>
   </property>
  </action>
  <action name="actionMapProperties">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-properties.png</normaloff>:/images/16x16/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Map &amp;Properties...</string>
   </property>
  </action>
  <action name="actionRecentFiles">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-open-recent.png</normaloff>:/images/16x16/document-open-recent.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Recent Files</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-save-as.png</normaloff>:/images/16x16/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionMoveLayerUp">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move Layer &amp;Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionMoveLayerDown">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move Layer Dow&amp;n</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select &amp;None</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionNewTileset">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;Tileset...</string>
   </property>
  </action>
  <action name="actionRemoveLayer">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Layer</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/window-close.png</normaloff>:/images/16x16/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAddTileLayer">
   <property name="text">
    <string>Add &amp;Tile Layer...</string>
   </property>
  </action>
  <action name="actionAddObjectLayer">
   <property name="text">
    <string>Add &amp;Object Layer...</string>
   </property>
  </action>
  <action name="actionDuplicateLayer">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/stock-duplicate-16.png</normaloff>:/images/16x16/stock-duplicate-16.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Duplicate Layer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionLayerProperties">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-properties.png</normaloff>:/images/16x16/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Layer &amp;Properties...</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/zoom-in.png</normaloff>:/images/16x16/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/zoom-out.png</normaloff>:/images/16x16/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomNormal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/zoom-original.png</normaloff>:/images/16x16/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::MapView</class>
   <extends>QGraphicsView</extends>
   <header>mapview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mapView</tabstop>
 </tabstops>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections/>
</ui>
